docs/docs/install/docker-compose/index.md

```diff
@@ -49,37 +49,56 @@
-
-2. Download the necessary files into the configuration directory
-
-    - Download the `docker-compose.yml` file from [https://github.com/centerofci/mathesar/raw/master/docker-compose.yml](https://github.com/centerofci/mathesar/raw/master/docker-compose.yml)
-     - Download the example `.env.example` file used for setting the [configuration variables](../configuration.md) from [https://github.com/centerofci/mathesar/raw/master/.env.example](https://github.com/centerofci/mathesar/raw/master/.env.example)
-     - Rename the downloaded `.env.example` to `.env`
-     - Download the `Caddyfile` from [https://github.com/centerofci/mathesar/raw/master/Caddyfile](https://github.com/centerofci/mathesar/raw/master/Caddyfile).
-
-3. Start the Mathesar web server
-    - Open the `.env` file in your favorite text editor.
-    - Add the **Database server configuration** environment variables to the `.env` file. Refer to the [Database Configuration Documentation](../configuration.md#database-configuration) and for information on the environment variables.
-    - Add the **Mathesar server configuration** environment variables to the `.env` file. Refer to the [Backend Documentation](../configuration.md#backend-configuration) and for information on the environment variables.
-    - Your `.env` file should look something like this
-      ``` 
-      .env
-      ===================================================================
-        POSTGRES_USER='mathesar'
-        POSTGRES_PASSWORD='mathesar'
-        POSTGRES_PORT='5432'
-        ALLOWED_HOSTS='https://<your_domain_name>'
-        SECRET_KEY='dee551f449ce300ee457d339dcee9682eb1d6f96b8f28feda5283aaa1a21'
-        DJANGO_DATABASE_KEY='default'
-        DJANGO_DATABASE_URL='postgresql://mathesar:mathesar@mathesar_db:5432/mathesar_django'
-        MATHESAR_DATABASES='(mathesar_tables|postgresql://mathesar:mathesar@mathesar_db:5432/mathesar)'
-        DJANGO_SUPERUSER_PASSWORD='password'
-      ```
-    - (Optional) The next step will start a Postgres database container called ` mathesar_db` automatically, this database won't be necessary if you already have a database, and you plan on using it. Refer to [Start Mathesar Server without the Database Server](#start-mathesar-server-without-the-database-server) for starting the web server without the additional db dependency.
-    - Start the Mathesar web server
+## Manual Install {#manual}
+
+1. Navigate to a directory where you'd like to store your Mathesar configuration. By convention, we do it within `/etc/mathesar`, but it can be a different directory if you like.
+
+    ```
+    sudo mkdir -p /etc/mathesar
+    cd /etc/mathesar
+    ```
+
+1. Download our [docker-compose.yml](https://github.com/centerofci/mathesar/raw/master/docker-compose.yml), and [.env.example](https://github.com/centerofci/mathesar/raw/master/.env.example) files to your configuration directory.
```

Comment:
@seancolsen Is there a reason why the items are not numbered correctly and instead are using `1` for every item?